src/rubocop/RubocopRunner.js

```diff
@@ -0,0 +1,55 @@
+'use babel'
+
+import { exec, findAsync } from 'atom-linter'
+
+const CFG_FILE = '.rubocop.yml'
```

Comment:
From [the documentation](https://rubocop.readthedocs.io/en/latest/configuration/#config-file-locations) it seems there are several locations where the configuration file can potentially be found.

Is it possible to let `rubocop` itself handle this for us? If not, do we need to handle the `~/.config/rubocop/config.yml` possibility? Note that you can pass an `Array` here and `findAsync` will search for each possibility.

Reply:
I think If the linter is intended to run on the project scope we should not check for the Rubocop configuration files outside that scope.